### DISCUSSION TITLE
Cookies- If I use Github Pages, am I using cookies?

### DISCUSSION THREAD

**Discussion by f4fsabadell** [community/community#22757] [Other Feature Feedback, Questions, & Ideas]
<p>Hii, I am doing a website, and I do not collect any data of the users, and I am not interested in collecting any type of it. So my question is, if I use Github pages, does Github collect data? Do I need to have a Cookies setting?</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/f4fsabadell">@f4fsabadell</a>, welcome to the GitHub Support Community!</p>
<p>No, not unless you’ve coded your site to use cookies! If you’re using a 3rd party theme or template then I’d recommend you check with the template developer, but GitHub itself doesn’t do anything to inject or use cookies on your site.</p>
